src/Module/clearall

```diff
@@ -0,0 +1 @@
+export default (tasks) => tasks.filter((task) => !task.completed);
```

Comment:
- [ ] when marking several tasks as completed and trying to remove the completed tasks this also removes tasks which weren't marked as completed, please fix this.